litmus-portal/backend/graphql-server/pkg/cluster/jwt.go

```diff
@@ -0,0 +1,60 @@
+package cluster
+
+import (
+	"errors"
+	"fmt"
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/util"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+var secret = ""
+
+func ClusterCreateJWT(id string) (string, error) {
+	initSecret()
+	expirationTime := time.Now().Add(12 * time.Hour)
+	claims := jwt.MapClaims{}
+	claims["cluster_id"] = id
+	claims["exp"] = expirationTime.Unix()
```

Comment:
```suggestion
	claims["expire_time"] = expirationTime.Unix()
```

Reply:
`exp` is a standard jwt claim, that's why I used that.